src/app/header/header.component.scss

```diff
@@ -1,18 +1,136 @@
+@import 'src/styles/styles';
+
 .header {
+  display: flex;
+  flex-direction: column;
+
   &__top {
-    display: flex;
-    justify-content: space-between;
+    @include flex-between;
+    padding: 16px 0;
+    border-bottom: 1px solid rgba(21, 21, 21, .05);
   }
 
   &__center {
+    @include flex-between;
+    padding: 13px 0 10px;
+  }
+
+  .list {
+    @include flex-between;
+
+    &__item {
+      &:not(:last-child) {
+        margin-right: 35px;
+      }
+    }
+  }
+
+  &__categories {
     display: flex;
-    justify-content: space-between;
+    align-items: center;
+    border: 1px solid $base-border-grey;
+    border-radius: $base-radius;
+    padding: 10px 16px;
+    background: $base-snow;
+
+    & hr {
+      border: none;
+      border-left: 1px solid $base-border-grey;
+      height: 20px;
+      width: 1px;
+      margin: 0 24px;
+    }
+
+    & img {
+      width: 9px;
+      margin-left: 8px;
+    }
   }
 
-  &__bottom {
-    & ul {
+  &__search {
+    width: 285px;
+    height: 100%;
+    @include flex-between;
+
+    & input {
+      width: 100%;
+      height: 100%;
+
+      background-color: $base-snow;
+
+      &::placeholder {
+        font-size: 14px;
+        font-weight: $font-weight-regular;
+        color: $base-light-grey;
+      }
+    }
+
+    & img {
+      width: 16px;
+    }
+  }
+
+  .dropdown {
+    position: relative;
+
+    &__label {
       display: flex;
-      justify-content: space-around;
+      align-items: center;
+      font-family: $base-font-secondary;
+      font-weight: $font-weight-semi-bold;
+      font-size: 15px;
+      color: $base-black;
+    }
+
+    &__list {
+      display: none;
```

Comment:
This logic with dropdown selection should be implemented with 
```ts 
<select formControlName="...">
        <option *ngFor="let category of categories" [value]="category">
          {{ category }}
        </option>
</select>
```
That would be in the future, when you will be implementing `Angular Reactive Forms`, so for now you can skip this. 
**Note**: Always remember, that when you want to hide important information and you use `display: none` or `visibility: hidden` or  other styles, every user can easily open devTools and display it.